src/Http/Header/Fields/UserAgent.php

```diff
@@ -20,15 +20,10 @@
  */
 class UserAgent implements HeaderField
 {
-    /** @var string */
-    private $userAgent;
-
-    /** @var string */
-    public const default = "Artemeon/HttpClient/Guzzle6";
+    private string $userAgent;
+    public const DEFAULT = "Artemeon/HttpClient/Guzzle6";
```

Comment:
ohne wieder eine Grundsatzdiskussion aufmachen zu wollen, aber ich behaupte mal, dass in mind. 99% unserer Code Base die Reihenfolge innerhalb des Class Body immer _Traits_, _Constants_, _Properties_, _Constructor(s)_ und _Methods_ ist 🤓

Reply:
So weit ich weiß kann man das den über die Config der IDE mitgeben. Dann würde das beim Format automatisch geordnet werden.

Reply:
<img width="999" alt="Screenshot 2021-01-06 at 11 29 37" src="https://user-images.githubusercontent.com/42121690/103758428-7b1c0780-5012-11eb-8b6c-824f51a14ac5.png">
ja das kann man 😉